ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/HD/LMDB.hs

```diff
@@ -306,24 +348,31 @@ initFromVals ::
   -> Db m l
   -> m ()
 initFromVals dbsSeq vals Db{..} = liftIO $ LMDB.readWriteTransaction dbEnv $
-  withDbSettingsRWMaybeNull dbSettings $ \case
+  withDbStateRWMaybeNull dbState $ \case
     Nothing -> zipLedgerTablesA initLMDBTable dbBackingTables vals $> ((), DbState{dbsSeq})
     Just _ -> Exn.throw $ DbErrStr "initFromVals: db already had state"
 
-initFromLMDBs :: (MonadIO m, IOLike m)
+initFromLMDBs ::
+     (MonadIO m, IOLike m)
   => Trace.Tracer m TraceDb
+  -> LMDBLimits
   -> FS.SomeHasFS m
   -> FS.FsPath
   -> FS.FsPath
   -> m ()
-initFromLMDBs tracer shfs from0 to0 = do
-  Trace.traceWith tracer $ TDBInitialisingFromLMDB from0
-  from <- guardDbDir GDDMustExist shfs from0
-  to <- guardDbDir GDDMustNotExist shfs to0
-  bracket
-    (liftIO $ LMDB.openEnvironment from defaultLMDBLimits) -- TODO assess limits, in particular this could fail if the db is too big
```

Comment:
So if I understand this correctly, before this PR we were using two limits: the one passed to `newLMDBBackingStore`, and `defaultLMDBLimits`. Now we're using the same limits, which are the ones passed to the `newLMDBBackingStore` function, right?